### ISSUE TITLE
CS2 demo crashes parser with probably nil dereference

### ISSUE THREAD

**Issue by sirherrbatka** [markus-wa/demoinfocs-golang#411]
**Describe the bug**
This demo will crash parser with runtime error: invalid memory address or nil pointer dereference error
On stack trace, I noticed:
github.com/markus-wa/demoinfocs-golang/v4/pkg/demoinfocs.(*parser).bindBomb.func1.4({{0x0, 0x0, 0x0}, 0x0, 0x0, {0x0, 0x0, 0x0}, {0x8bb1a8, 0x0}, ...})
And sure enough, those pointers look null to me.

**To Reproduce**
Get this file https://antiochia.duckdns.org/index.php/s/qARR64pwXbeM65q

Code:
```
package main

import (
	dem "github.com/markus-wa/demoinfocs-golang/v4/pkg/demoinfocs"
	"os"
	"fmt"
)

func main() {
	f, e := os.Open("/home/shka/Pobrane/match730_003636915017762210008_1288756472_183.dem") // replace this
	fmt.Println(e)

	p := dem.NewParser(f)
	error := p.ParseToEnd()
	if error != nil {
		panic(error)
	}
}
```

**Expected behavior**
Not crash.

**Library version**
v4

**Additional context**
Running on x86_64 Linux.

**Comment by markus-wa** [comment]
I will close this and mark it as fixed given #412 has been merged

please comment and re-open if the issue persists :slightly_smiling_face:

**Comment by sirherrbatka** [comment]
@markus-wa well, this demo will still crash parser, but in a different way (Unexpected EOF).
